_posts/2019-10-18-parent-instance-error.md

```diff
@@ -0,0 +1,15 @@
+---
+site: freiburg
+tags: [devops, bug]
+title: Parent instance error
+location: Freiburg, Germany
+---
+
+UseGalaxy.eu is currently facing some issues with tools execution and may take a while to find a solution.  
+If you get an error message of this kind:  
+`Parent instance <ToolShedRepository at 0x7fe3f20dca10> is not bound to a Session; lazy load operation of attribute 'tool_dependencies' cannot proceed (Background on this error at: http://sqlalche.me/e/bhk3)`
+
+we suggest trying to rerun your failed job.  
```

Comment:
... until its execution succeeds.
This may also be the perfect time to start enjoying your weekend!